imports/services/BoardService.ts

```diff
@@ -1,8 +1,9 @@
 import { IBoard } from '../excalidraw/types'
-import { BoardCollection, BoardRecord } from '../models'
+import { BoardCollection, BoardRecord, BoardFavoritesCollection } from '../models'
 import { Collections } from '../models/Collections'
 import { BaseService } from './BaseService'
 import { Meteor } from 'meteor/meteor'
+import { _makeBoard, _makeBoards } from '../subscriptions/utils'
```

Comment:
we shoun't import funcitons from client directly. 

can you please move these 2 functions into a common folder, like `utils/boards.ts`